.gitignore

```diff
@@ -35,6 +35,7 @@ build/Release
 # Dependency directories
 node_modules/
 jspm_packages/
+.idea/
```

Comment:
What is the use of this directory? Is it editor-specific?

Reply:
yes, webstrom specific